packages/@aws-cdk/aws-iot-actions-alpha/test/kinesis-firehose/integ.firehose-put-record-action.js.snapshot/test-stack.template.json

```diff
@@ -273,7 +273,7 @@
     "FirehoseCidrBlock": "35.158.127.160/27"
    },
    "eu-central-2": {
-    "FirehoseCidrBlock": "16.62.183.32/27"
+    "FirehoseCidrBlock": "16.62.183.64/26/27"
```

Comment:
This doesn't look right

Reply:
Same for `eu-south-2` by the way, although that's not your PR

Reply:
And `me-central-1` and a bunch of others. I think we need to refactor how the firehose cidr block fact works.